tmt/steps/provision/__init__.py

```diff
@@ -459,22 +459,32 @@ def _export_environment(self, execute_environment=None):
         # override environment provided to execute().
         environment.update(self.parent.plan.environment)
         # Prepend with export and run as a separate command.
```

Comment:
The comment is now irrelevant. Also the method name does not match the updated implementation. Let's keep the old name `_export_environment` for providing the `export VAR=VAL` output and rename this method to `_prepare_environment`.